.travis.yml

```diff
@@ -24,4 +24,4 @@ before_script:
   - python manage.py migrate
 
 script:
-  - python manage.py test
+  - debug=true python manage.py test
```

Comment:
is this change intentional/needed?

Reply:
yep - for some reason this is needed for any of the tests which render curricula views to pass on travis.